<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConnections</class>
 <widget class="QDialog" name="DialogConnections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>348</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>419</width>
        <height>277</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="LayoutConnections">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="BtnInterface">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Show interface</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="BtnClose">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LabelSource">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="LabelTarget">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogConnections</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
